### PR TITLE
Normalize sql queries

### PR DESCRIPTION
**Pull request by rmarianski**
This doesn't address the problem with handling table sources between the sql and python paths, so the sql calculation is likely to be broken in this pr.

### CODE REVIEW

File: queries/boundaries.jinja2 [hunk 1]
```diff
@@ -101,16 +100,14 @@ UNION ALL
...
-  hstore(ARRAY[
-           'source', 'openstreetmap.org',
-           'kind', 'maritime']) AS mz_additional_tags,
-  NULL::json AS tags
+  json_build_object('kind', 'maritime') as mz_properties,
+  jsonb_build_object(
+    'source', 'openstreetmap.org',
+    'min_zoom', 8,
+    'maritime_boundary', true,
+    'kind', 'maritime'
```
Comment (zerebubuth) [inline queries/boundaries.jinja2]: Does `kind` need to be in both `__properties__` and `mz_properties`? Could we move `maritime_boundary` into `mz_properties`?

Comment (rmarianski) [inline queries/boundaries.jinja2]: I only left `mz_properties` there for testing purposes to verify that the output was the same as from the database. I'll remove it from the queries when we're ready to merge.

Comment (zerebubuth) [inline queries/boundaries.jinja2]: Ah, cool. Sounds good :+1:

File: yaml/pois.yaml [hunk 2]
```diff
@@ -3,6 +3,25 @@
...
+    area: {col: area}
+    protect_class: {col: protect_class}
+    operator: {col: operator}
+    religion: {col: religion}
+    sport: {col: sport}
+    mz_transit_score: {col: mz_transit_score}
+    mz_transit_root_relation_id: {col: mz_transit_root_relation_id}
+    train_routes: {col: train_routes}
+    subway_routes: {col: subway_routes}
+    light_rail_routes: {col: light_rail_routes}
```
Comment (nvkelso) [inline yaml/pois.yaml]: Missing:

`tram_routes: {col: tram_routes}`

As tye JINJA has:

`'tram_routes', (transit_routes).tram_routes`

File: queries/places.jinja2 [hunk 3]
```diff
@@ -65,23 +54,20 @@ WHERE
...
+    {% filter geometry %}label_position{% endfilter %} AS __geometry__,
+    json_build_object(
+      'kind', wof_np.placetype_string
+    ) AS mz_properties,
+    to_jsonb(l10n_name) || jsonb_build_object(
+      'name', name,
+      'source', 'whosonfirst.mapzen.com',
+      'mz_n_photos', n_photos,
+      'area', area::bigint,
+      'max_zoom', max_zoom,
```
Comment (nvkelso) [inline queries/places.jinja2]: Missing:

`'min_zoom', min_zoom,`

File: yaml/roads.yaml [hunk 4]
```diff
@@ -1,5 +1,11 @@
 global:
+  - &output_properties
+    source: {col: source}
```
Comment (nvkelso) [inline yaml/roads.yaml]: Surprised `name: { col: name }` isn't here in the global section.

Comment (nvkelso) [inline yaml/roads.yaml]: Is that due to the synthetic_columns, is that necessary anymore? 

```
synthetic_columns:
   - name
```

File: yaml/pois.yaml (hunk 2 above)
Comment (rmarianski) [inline yaml/pois.yaml]: Good catch! Fixed in 95d993c.

File: queries/places.jinja2 (hunk 3 above)
Comment (rmarianski) [inline queries/places.jinja2]: Good catch! Fixed in 710e0bd.

File: yaml/roads.yaml (hunk 4 above)
Comment (rmarianski) [inline yaml/roads.yaml]: Do we want pistes to have names? It looks like now they don't. They have `piste_name` and `route_name`.

Comment (nvkelso) [inline yaml/roads.yaml]: Ah, right. Maybe add a comment up in the global section noting that? Something like

```
    # not all features (e.g. pistes) include names so names are added below
```

Comment (rmarianski) [inline yaml/roads.yaml]: Good call. Added in dd9c991.

**Pull request merged**